lbrynet/core/LBRYWallet.py

```diff
@@ -315,24 +370,18 @@ def get_claim_for_name(claims):
         return d
 
     def _get_stream_info_from_value(self, result, name):
-        r_dict = {}
         if 'value' in result:
             value = result['value']
 
             try:
                 value_dict = json.loads(value)
             except (ValueError, TypeError):
                 return Failure(InvalidStreamInfoError(name))
-            r_dict['sources'] = value_dict['sources']
-            for field in BASE_METADATA_FIELDS:
-                    r_dict[field] = value_dict[field]
-            for field in value_dict:
-                if field in OPTIONAL_METADATA_FIELDS:
-                    r_dict[field] = value_dict[field]
-
+            m = Metadata(value_dict)
         if 'txid' in result:
-            d = self._save_name_metadata(name, r_dict['sources']['lbry_sd_hash'], str(result['txid']))
-            d.addCallback(lambda _: r_dict)
+            d = self._save_name_metadata(name, str(result['txid']), m['sources']['lbry_sd_hash'])
+            d.addCallback(lambda _: log.info("lbry://%s complies with %s" % (name, m.metaversion)))
```

Comment:
log will do the string interpolation for you `log.info("lbry://%s complies with %s", name, m.metaversion)`